### PR TITLE
New package: reaper-6.19.

### PR DESCRIPTION
**Pull request by fosslinux**
A very common DAW used professionally.

Is nonfree, but allows redistribution, AFAICT.

### CODE REVIEW

File: srcpkgs/reaper/files/application-x-reaper.xml [hunk 1]
```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (ericonr) [inline srcpkgs/reaper/files/application-x-reaper.xml:1]: I think you forgot to install this, tho I kinda hate the idea of carrying it :/

Comment (fosslinux) [reply srcpkgs/reaper/files/application-x-reaper.xml:1]: Oh, whoops, no I forgot to remove it; this is installed by the install script but is unnessecary, update-mime-database generates basically this.

**Comment by fosslinux** [comment]
Removed extraneous file

**Comment by fosslinux** [comment]
Updated to 6.18.

**Comment by fosslinux** [comment]
Updated to 6.19. @ericonr ping

**Comment by ericonr** [comment]
Is vendoring the desktop file really necessary? I really dislike doing that, though I can understand the need with proprietary stuff :/

**Comment by fosslinux** [comment]
Yeah it kinda is, the alternative is not having it in the desktop menus.

**Comment by biopsin** [comment]
I've posted a request on https://forum.cockos.com/showthread.php?p=2428322#post2428322 to include desktop file in tarballs..

**Comment by fosslinux** [comment]
Yeah, unfortunately upstreams regularly don't care too much about distros "use our stuff"!111

**Comment by fosslinux** [comment]
Yeah, unfortunately upstreams regularly don't care too much about distros "use our stuff"!111

**Comment by biopsin** [comment]
@fosslinux anyways thank you for template, I use Reaper and I don't mind repacking it locally if it does'nt end up in the repo. I still have to pkg Swell for gtk+ anyways so I lose/gain not that much.

**Comment by ericonr** [comment]
> Download REAPER below for a free, fully functional 60-day evaluation.

Is this enforced?

https://www.reaper.fm/dist-agreement.php

I think you have to sign up to become a distributor?

**Comment by fosslinux** [comment]
Aargh, I had not found that earlier. You seem to be correct. We cannot distribute this. `restricted` time... (or we can just not merge, I don't mind either way).

**Comment by timsofteng** [comment]
Hello!
Any updates?

**Comment by fosslinux** [comment]
We cannot put this in the official repository. You are free to use xbps-src on this PR. I will also update this pr with `restricted=yes` and hopefully it can be merged.

**Comment by timsofteng** [comment]
@fosslinux got it.
Hopefully it will be added into xbps-src templates.

**Comment by timsofteng** [comment]
.

**Pull request closed without merging**